app/src/test/java/com/sefford/artdrian/data/dto/deserializers/WallpaperResponseDeserializerTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.sefford.artdrian.data.dto.deserializers
+
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import com.karumi.kotlinsnapshot.matchWithSnapshot
+import com.sefford.artdrian.datasources.WallpaperApi
+import com.sefford.artdrian.model.Metadata
+import com.sefford.utils.Files
+import io.kotest.inspectors.runTest
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.jupiter.api.Assertions.*
+
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class WallpaperResponseDeserializerTest : Files {
+
+    private val gson : Gson = GsonBuilder()
+        .registerTypeAdapter(WallpaperResponse::class.java, WallpaperResponseDeserializer())
+        .create()
+
+    @Test
+    fun deserialize() {
```

Comment:
Is it ok this test for the deserializer? 🤔

Reply:
You know I am very succint for testing names. It's just a name, but can be improved. Maybe we can talk about this on our next 1:1.